libs/stats/odc/stats/tasks.py

```diff
@@ -238,15 +263,19 @@ def msg_default(msg):
 
 
 class TaskReader:
-    def __init__(self,
-                 cache: Union[str, DatasetCache],
-                 product: Optional[OutputProduct] = None):
+    def __init__(
+        self, cache: Union[str, DatasetCache], product: Optional[OutputProduct] = None
+    ):
+        self.cache_path = None
```

Comment:
should be `self._cache_path` this is private property.

Reply:
Fixed.